packages/next-templates/src/app/wmebv/ingelogd/stap1/page.tsx

```diff
@@ -54,7 +54,7 @@ export default function home() {
 
   return (
     <UtrechtPage>
-      <ExampleHeaderFunnelWmebv userURL={userdata.userURL} username={userdata.username} />
+      <ExampleHeaderFunnelWmebv UserChevronIcon userURL={userdata.userURL} username={userdata.username} />
```

Comment:
Ik zou `UserChevronIcon` niet met een hoofdletter doen, maar als `userChevronIcon` omdat het geen component is maar een attribuut